SPECS/mstflint/mstflint.spec

```
@@ -40,13 +56,33 @@ find . -type f -iname '*.cpp' -exec chmod a-x '{}' ';'
 %configure --enable-fw-mgr --enable-adb-generic-tools
 %make_build
 
+# Compile mstflint driver
+export EXTRA_CFLAGS='-DVERSION=\"%version\"'
+cd kernel
+set -- *
+mkdir source
+mv "$@" source/
```

Why do we need to overwrite the positional parameters? Wouldn't this work as well:
```suggestion
mkdir source
mv * source/
```

---

Removed.